docs/product-updates/2023-06-07-api-branch-management.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New, Bump API]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+[Branching at Bump.sh](https://docs.bump.sh/help/branching/) makes your API version workflow easier.
+
+## Latest release
+
+And now, Branch Management is at hand with the [Bump API branching endpoints](https://developers.bump.sh/group/endpoint-branches).
+Designed to enhance your API development process, these new endpoints gives you the opportunity to boost your CI deployments. With the introduction of branch creation, listing, setting default branch, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions.
```

Comment:
Should we use `revolutionizes`? It feels a bit overpromised. Maybe we could replace by something like `optimize`? WDYT?